### DISCUSSION TITLE
Item only progression mode

### DISCUSSION THREAD

**Discussion by Eliteshiningnite** [Malcolm-Q/LC-LateGameUpgrades#538] [Ideas]
Please add a config option to remove upgrades from the store all-together while still being able to make progress with defined items in the config. I was trying every option in the config but it seems to be if an upgrade isn't in the shop you can't make progress towards it. Wanted to make it item only progression for a slower power scaling with slight randomness like only getting progress for beekeeper by selling bee hives but it just doesn't seem possible.

**Comment by WhiteSpike** [comment]
So the suggestion is to make a toggle which makes the upgrades not purchaseable when the mode is on? Just to make things clear.

**Reply by Eliteshiningnite** [reply]
Yea that's pretty much what I mean. I tried to make my point too complicated

**Reply by WhiteSpike** [reply]
I am thinking of maybe make the store still show the upgrades but you won't have the option of purchasing it. Just so you can still see the progress on them when you're using modes like UniqueScrap along with what level it's at and to see what it gives.

Would this work for your needs or not?

**Reply by Eliteshiningnite** [reply]
Sorry for the late reply, that would be great.
